src/RestAPI/dlwsrestapi.py

```diff
@@ -314,15 +314,17 @@ def __init__(self):
         self.get_parser = reqparse.RequestParser()
         self.get_parser.add_argument("jobId", required=True)
         self.get_parser.add_argument("userName", required=True)
-        self.get_parser.add_argument("resourcegpu", required=True)
+        self.get_parser.add_argument("mingpu", required=True)
+        self.get_parser.add_argument("maxgpu", required=True)
 
     def get(self):
```

Comment:
Shall we use `POST` to do modifications?

Reply:
ok